sample_factory/algo/learning/tests/test_rnn.py

```diff
@@ -43,27 +44,20 @@ def check_packed_version_matching_loopy_version(self, T, N, D, random_dones, nor
 
             norm = torch.norm(packed_out - loopy_out)
             print(norm)
-            self.assertLess(norm, norm_tolerance)
-            self.assertTrue(np.allclose(packed_out.detach().numpy(), loopy_out.detach().numpy(), atol=2e-6))
-
-    def test_full_with_larger_param(self):
-        T = 37  # recurrence, bptt
-        N = 64  # batch size
-        D = 42  # RNN cell size (hidden state size)
-        self.check_packed_version_matching_loopy_version(T, N, D, True, 9e-6)
-        self.check_packed_version_matching_loopy_version(T, N, D, False, 9e-6)
-
-    def test_full(self):
-        T = 27  # recurrence, bptt
-        N = 64  # batch size
-        D = 10  # RNN cell size (hidden state size)
-        self.check_packed_version_matching_loopy_version(T, N, D, True)
-        self.check_packed_version_matching_loopy_version(T, N, D, False)
-
-    def test_trivial(self):
-        T = 5  # recurrence, bptt
-        N = 1  # batch size
-        D = 1  # RNN cell size (hidden state size)
-        self.check_packed_version_matching_loopy_version(T, N, D, True)
-        self.check_packed_version_matching_loopy_version(T, N, D, False)
-
+            assert norm < norm_tolerance
+            assert np.allclose(packed_out.detach().numpy(), loopy_out.detach().numpy(), atol=2e-6)
+
+    @pytest.mark.parametrize("T", [37])
```

Comment:
Ok please use single quotes. I will try to set up the automatic formatter after I finish my current tasks.

Reply:
Ok, sounds good.